content/en/docs/concepts/policy/limit-range.md

```diff
@@ -56,337 +56,14 @@ there may be contention for resources. In this case, the Containers or Pods will
-
-{{< codenew file="admin/resource/limit-memory-ratio-pod.yaml" >}}
-
-```shell
-kubectl apply -f https://k8s.io/examples/admin/resource/limit-memory-ratio-pod.yaml
-```
-
-Describe the <limit-memory-ratio-pod> LimitRange with the following kubectl command:
-
-```shell
-kubectl describe limitrange/limit-memory-ratio-pod
-```
-
-```shell
-Name:       limit-memory-ratio-pod
-Namespace:  limitrange-demo
-Type        Resource  Min  Max  Default Request  Default Limit  Max Limit/Request Ratio
-----        --------  ---  ---  ---------------  -------------  -----------------------
-Pod         memory    -    -    -                -              2
-```
-
-Create a pod with `requests.memory=100Mi` and `limits.memory=300Mi`:
-
-{{< codenew file="admin/resource/limit-range-pod-3.yaml" >}}
-
-```shell
-kubectl apply -f https://k8s.io/examples/admin/resource/limit-range-pod-3.yaml
-```
-
-The pod creation failed as the ratio here (`3`) is greater than the enforced limit (`2`) in `limit-memory-ratio-pod` LimitRange:
-
-```
-Error from server (Forbidden): error when creating "limit-range-pod-3.yaml": pods "busybox3" is forbidden: memory max limit to request ratio per Pod is 2, but provided ratio is 3.000000.
-```
-
-## Clean up
-
-Delete the `limitrange-demo` namespace to free all resources:
-
-```shell
-kubectl delete ns limitrange-demo
-```
-Change your context to `default` namespace with the following command:
-
-```shell
-kubectl config set-context --current --namespace=default
-```
-
 ## Examples
 
-- See  [a tutorial on how to limit compute resources per namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/) .
-- Check [how to limit storage consumption](/docs/tasks/administer-cluster/limit-storage-consumption/#limitrange-to-limit-requests-for-storage).
-- See a [detailed example on quota per namespace](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/).
+- See [how to configure minimum and maximum CPU constraints per namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/).
+- See [how to configure minimum and maximum Memory constraints per namespace](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/).
+- See [how to configure default CPU Requests and Limits per namespace](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/).
+- See [how to configure default Memory Requests and Limits per namespace](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/).
+- Check [how to configure minimum and maximum Storage consumption per namespace](/docs/tasks/administer-cluster/limit-storage-consumption/#limitrange-to-limit-requests-for-storage).
+- See a [detailed example on configuring quota per namespace](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/).
```

Comment:
This looks OK and makes the page better. In my view, it would be more common to put those links into the “What's Next” section.

Aside: if you want to preserve the `#examples` fragment identifier, you can add an HTML element such as:
```html
<div id="examples" />
```

Reply:
Thanks for the review! Not sure what you mean with the last comment about fragment identifier. I though Hugo was responsible to generate those elements.